tests/unit/multi_select_test.coffee

```diff
@@ -5,39 +7,43 @@ moduleForComponent 'multi-select', '[Unit] Multi select component',
     'template:select-item-layout'
     'template:select-item'
   ]
+  teardown: ->
+    Ember.run ->
+      multiSelect.destroy()
+    multiSelect = null
```

Comment:
`Ember.run` doesn't schedule anything, it executes the function inside a run loop.

So the execution order isn't changed.